axelor-account/src/main/java/com/axelor/apps/account/service/ReimbursementImportService.java

```diff
@@ -79,11 +84,13 @@ public void runReimbursementImport(Company company) throws AxelorException, IOEx
     this.testCompanyField(company);
 
     AccountConfig accountConfig = company.getAccountConfig();
+    String dataUploadDir = appService.getFileUploadDir();
```

Comment:
Move this line of code to starting of the method
and check with append File.separator in same line of code